src/styles/common/_buttons.scss

```diff
@@ -86,14 +86,22 @@
     }
 }
 
+.btn-plain {
+    margin: 0;
+    padding: 0;
+
+    // prevent default .btn behavior of adding space between buttons
+    & + & {
+        margin: 0;
```

Comment:
We should try to limit overrides as much as possible. Can this just override `margin-left`?